test/unit/ngsi-ld/examples/contextRequests/updateContextAliasPlugin7.json

```diff
@@ -6,10 +6,7 @@
             "type": "Property",
             "value": {
                 "@type": "Array",
-                "@value": [
-                    "iot",
-                    "device"
-                ]
+                "@value": "[\"iot\",\"device\"]"
             }
         },
         "type": "Light"
```

Comment:
I suggest to modify the data sent instead of the expectation. Makes more sense. https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/ngsi-ld/plugins/alias-plugin_test.js#L379

Reply:
NTC. its a ngsi-ld test